packages/fxa-admin-panel/src/components/AccountSearch/index.tsx

```diff
@@ -49,20 +49,82 @@ export const GET_ACCOUNT_BY_EMAIL = gql`
   }
 `;
 
+export const GET_EMAILS = gql`
+  query fetchAll($search: String!) {
+    getEmailsLike(search: $search) {
+      email
+    }
+  }
+`;
+
 export const AccountSearch = () => {
   const [inputValue, setInputValue] = useState<string>('');
   const [showResult, setShowResult] = useState<Boolean>(false);
+  const [suggestions, setSuggestion] = useState<Array<String>>([]);
```

Comment:
this should be a lower case `string`. in fact `string[]` is more idiomatic than `Array<string>`